code/drasil-gool/lib/Drasil/GOOL/InterfaceCommon.hs

```diff
@@ -349,6 +357,7 @@ type MSStatement a = MS (a (Statement a))
 class (ValueSym r) => StatementSym r where
   type Statement r
   valStmt :: SValue r -> MSStatement r -- converts value to statement
+  emptyValStmt :: MSStatement r
```

Comment:
Ah, after looking more into it I see my assumptions weren't quite right.  `G.emptyStmt` isn't exported by `Drasil.GOOL`, and anyway it requires an instance of `CommonRenderSym`, which is internal to GOOL.

I also wasn't thinking about the `CodeInfo` instances.  Since they don't instantiate `CommonRenderSym`, their implementations _are_ different.

Honestly, I was just surprised we didn't already have something like this.  I think if we need access to an empty statement in `drasil-code`, this is the best way to do it.

I did have one other thought.  I'm not exactly sure if it would work, but could we just have [exc](https://github.com/JacquesCarette/Drasil/blob/fc1e8e48e80c40a4990f47930a2499cbe6342aea/code/drasil-code/lib/Language/Drasil/Code/Imperative/Modules.hs#L377-L384) return `[]` instead of `[emptyValStmt]`?  I'm just thinking that if we can get away with not extending GOOL when there's another way, it might be nice.

Reply:
I agree